### ISSUE TITLE
`Lint/FloatComparison` should detect `case` statements

### ISSUE THREAD

**Issue by fatkodima** [rubocop/rubocop#13194]
This is currently not detected:

```ruby
# bad
case value
when 1.0  <-- float comparison
  foo
when 2.0  <-- float comparison
  bar
end
```

**Comment by Earlopain** [comment]
What would the good case look like here? Move to if/else with tolerance, or pattern matching somehow?

**Comment by fatkodima** [comment]
if/else I think.

**Comment by daisuke** [comment]
I'd be happy to contribute a fix for this issue if no one is working on it yet.
